test/rules/Lambda.test.ts

```diff
@@ -351,4 +353,29 @@ describe('AWS Lambda', () => {
       validateStack(stack, ruleId, TestType.VALIDATION_FAILURE);
     });
   });
+
+  describe('LambdaLogging: Ensure that Lambda functions have a corresponding Log Group', () => {
```

Comment:
General comment, also add some tests using the `L2` construct (if possible). There are some cases ([example](https://github.com/cdklabs/cdk-nag/blob/main/src/rules/opensearch/OpenSearchNoUnsignedOrAnonymousAccess.ts#L42)) where the CDK uses custom resources/alternative patterns to achieve the same functionality. Having tests that use L2 constructs help us catch some of these problems